user_modules/script_runner.lua

```diff
@@ -5,9 +5,11 @@ local commonFunctions = require('user_modules/shared_testcases/commonFunctions')
 
 local isInitialStep = true
 local isPrintTitle = false
+local isSelfIncluded = true
```

Comment:
@dboltovskyi I would propose to add `isSelfIncluded` flag into `runner.testSettings` table instead, because the flag is a part of settings of current test, not `runner`.

Also function runner.SetParameters is redundant if use `runner.testSettings`

Reply:
@aderiabin Agreed. Done in commit 391ab59